<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quiz</string>
  </property>
  <widget class="QLabel" name="quiz">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> </string>
   </property>
  </widget>
  <widget class="QGroupBox" name="optionGroup">
   <property name="geometry">
    <rect>
      <x>100</x>
      <y>70</y>
      <width>300</width>
      <height>150</height>
    </rect>
  </property>
   <property name="title">
      <string>선택지</string>
    </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
      <widget class="QRadioButton" name="groupBox_rad1"/>
    </item>
    <item>
      <widget class="QRadioButton" name="groupBox_rad2"/>
    </item>
    <item>
      <widget class="QRadioButton" name="groupBox_rad3"/>
    </item>
    <item>
      <widget class="QRadioButton" name="groupBox_rad4"/>
    </item>
  </layout>
  </widget>
  <widget class="QPushButton" name="btn_1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>정답 확인</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>